### PR TITLE
Issue #PS-1389 feat: UI for displaying center wise attendance report as per Figma

### PR DESCRIPTION
**Pull request by Aar-if**
Issue #PS-1389 feat: UI for displaying center wise attendance report as per Figma

### CODE REVIEW

File: src/components/AttendanceComparison.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  { name: 'Koradi', Attendance: 70 },
+  { name: 'Neri', Attendance: 65 },
+  { name: 'Palsal', Attendance: 80 },
+  { name: 'Wadoda', Attendance: 90 },
+];
+
+const AttendanceComparison: React.FC = () => {
+  const { t } = useTranslation();
+  const [centerType, setCenterType] = useState('Regular');
+  const data = centerType === 'Regular' ? regularData : remoteData;
```
Comment (itsvick) [inline src/components/AttendanceComparison.tsx:29]: @Aar-if add the centerType in app.constant as enum and use here

File: src/components/AttendanceComparison.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        <Typography align="left">{t('DASHBOARD.BLOCK_AVERAGE_ATTENDANCE')}: 76%</Typography>
+        <ResponsiveContainer width="100%" height={400} >
+          <BarChart
+          
+            layout="vertical"
+            data={data}
+            margin={{ top: 5, right: 30, left: 20, bottom: 5 }}
+            
+          >
+            <CartesianGrid stroke="#f5f5f5" />
```
Comment (itsvick) [inline src/components/AttendanceComparison.tsx:64]: @Aar-if add color from theme variables

File: src/components/AttendanceComparison.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+];
+
+const remoteData = [
+  { name: 'Khasala', Attendance: 75 },
+  { name: 'Koradi', Attendance: 70 },
+  { name: 'Neri', Attendance: 65 },
+  { name: 'Palsal', Attendance: 80 },
+  { name: 'Wadoda', Attendance: 90 },
+];
+
```
Comment (itsvick) [inline src/components/AttendanceComparison.tsx:25]: @Aar-if write a different file to get the my-cohorts and then loop over it to get the percentage of it.. make it separate from the component file

Comment (Aar-if) [reply src/components/AttendanceComparison.tsx:25]: yes

**Pull request merged**